packages/core/src/browser/icons/spinner.svg

```diff
@@ -0,0 +1,6 @@
+<!--Licensed under the MIT License. See License.txt in the project root for license information.-->
```

Comment:
Any reason for using MIT as a license? The link "see license.txt" is bogus since no MIT license exists.

Reply:
All files have this header. Okay, will remove.

Reply:
> All files have this header. Okay, will remove.

Which files exactly?

Reply:
Another icon. I looked there to know how are they formatted and what root props they have.